buildprocess/configureWebpack.js

```diff
@@ -121,7 +121,8 @@ function configureWebpack(terriaJSBasePath, config, devMode, hot, MiniCssExtract
                         '@babel/proposal-class-properties',
                         '@babel/proposal-object-rest-spread',
                         'babel-plugin-styled-components',
-                        require.resolve('@babel/plugin-syntax-dynamic-import')
+                        require.resolve('@babel/plugin-syntax-dynamic-import'),
+                        '@babel/plugin-proposal-optional-chaining'
```

Comment:
preset-env already includes this on by default as of 7.8.0, and we've recently explicitly bumped it to 7.10.x

```suggestion
                        require.resolve('@babel/plugin-syntax-dynamic-import')
```

Reply:
See https://github.com/TerriaJS/terriajs/pull/4386#discussion_r466955192